package.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "cc-global",
+  "version": "1.0.0",
```

Let's start the version at `0.1.0`
```suggestion
  "version": "0.1.0",
```